lib/daru.rb

```diff
@@ -26,19 +26,20 @@ module Daru
     10 => 31,
     11 => 30,
     12 => 31
-  }
+  }.freeze
 
-  SPLIT_TOKEN = ','
-  class << self
-    @@lazy_update = false
+  @lazy_update = false
```

Comment:
Currently, almost ALL guides and tutorials explicitly bun class variable (in favor of "instance variable at module level"). Though, I'm not intending to fight for this one thing till the end of the world :) Chaning back.